submodules/groups/groups.js

```diff
@@ -2099,11 +2121,13 @@ define(function(require) {
 			delete callflow.metadata;
 
 			self.callApi({
-				resource: 'callflow.update',
+				resource: 'callflow.patch',
```

Comment:
As a best practice, I wouldn't recommend updating an existing request method as it might be used in other places, create new one instead (`groupsPatchCallflow` in this case).

```suggestion
				resource: 'callflow.update',
```

Reply:
thanks, done!